<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTextEdit" name="textEditIPServer">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>31</height>
           </size>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;127.0.0.1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_conectar">
            <property name="text">
             <string>Conectar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_desconectar">
            <property name="text">
             <string>Desconectar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Valor Mínimo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="sliderMin">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QLCDNumber" name="lcdNumber"/>
          </item>
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Valor Máximo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="sliderMax">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="lcdNumber_2"/>
          </item>
          <item row="2" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Tempo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="sliderTempo">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <widget class="QLCDNumber" name="lcdNumber_3">
            <property name="intValue" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_comecar">
            <property name="text">
             <string>Começar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_parar">
            <property name="text">
             <string>Parar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelConectado">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_conectar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>conectar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_desconectar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>desconectar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderTempo</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setMin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setMax()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_comecar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>comecar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_parar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>parar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderTempo</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>alteratempo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tcpConnect()</slot>
  <slot>conectar()</slot>
  <slot>desconectar()</slot>
  <slot>setMin()</slot>
  <slot>setMax()</slot>
  <slot>enviar()</slot>
  <slot>comecar()</slot>
  <slot>parar()</slot>
  <slot>alteratempo()</slot>
 </slots>
</ui>
